src/goto-checker/bmc_util.h

```diff
@@ -197,7 +197,8 @@ void run_property_decider(
   "(incremental-loop):" \
   "(unwind-min):" \
   "(unwind-max):" \
-  "(ignore-properties-before-unwind-min)"
+  "(ignore-properties-before-unwind-min)" \
+  "(symex-cache-dereferences)"
```

Comment:
Please also add a `\` to the last line (to avoid the pain you've had in that you had to touch two lines even though only one is new).

Reply:
Fixed.

Reply:
Fixed.